src/main/webapp/ChapMap.js

```diff
@@ -324,8 +225,9 @@ class ChapMap {
    * Retrieves markers from the server and adds them to the map
    * @param {firebase.idToken} idToken the current user's getIdToken
    */
-  getMarkers(idToken) {
-    fetch(`/map-server?idToken=${idToken}&idRoom=${roomId}`)
+  getMarkers(userIdToken) {
+    this.idToken = userIdToken;
```

Comment:
Is it possible for userIdToken to change in a session? I'm not sure I understand why we need to reset it whenever we fetch the markers.

Reply:
The user token changes whenever a user logs in but does not change afterwards (it identifies who is logged in at the moment). Since fetching the pre-existing markers for this map only occurs once when the map is loading, and this is the first time idToken is used to communicate with the server, I decided to initialize the instance variable here. Also, I need the IdToken later on whenever I send information to the server which is why I put it in an instance variable.

Reply:
I see. I thought this is called whenever we read markers. This is good then